examples/simpleapp/pom.xml

```diff
@@ -2,7 +2,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>com.horizen</groupId>
 	<artifactId>Sidechains-SDK-simpleapp</artifactId>
-	<version>0.2.0</version>
+	<version>0.2.0-SNAPSHOT</version>
```

Comment:
update the version + 0.1 to master/dev branches.

Reply:
done